cl/users/templates/includes/webhooks_htmx/webhooks-list.html

```diff
@@ -16,24 +16,26 @@
       </p>
     </td>
     <td>{{ webhook.url }}</td>
-    <td class="right" hx-headers='{"X-CSRFToken": "{{ csrf_token }}"}'>
+    <td class="right" hx-headers='{"X-CSRFToken": "{{ csrf_token }}"}' data-loading-states>
       <button hx-delete="{% url "webhooks-detail" pk=webhook.id format='json' %}"
               hx-confirm="Are you sure you want to delete this Webhook?"
-              class="btn btn-danger btn-xs">
-        <i class="fa fa-trash-o"></i>&nbsp;Delete</button>
+              class="btn btn-danger btn-xs" data-loading-disable>
+        <i class="fa fa-trash-o"></i>&nbsp;Delete
+        <i data-loading class="fa fa-spinner fa-spin"></i>
+      </button>
       <button class="btn btn-primary btn-xs"
               hx-get="{% url "webhooks-detail" pk=webhook.id format='html' %}"
-              hx-target="#webhooks-body">
+              hx-target="#webhooks-body" data-loading-disable>
         <i class="fa fa-pencil"></i>&nbsp;Edit</button>
```

Comment:
I liked that when clicking the Edit button it's disabled together with the Delete button (that shows the spinner). It might be also good to show the spinner inside the Edit button. 

If possible something like this:
- Clicking Edit button: 
 Disable the Delete and Edit buttons, only show the spinner inside the Edit button.
- Clicking Delete button: 
 Disable the Delete and Edit buttons, only show the spinner inside the Delete button.

Reply:
@albertisfu The UI would look like this if we change the edit and delete buttons:

![isolate-edit-and-delete](https://user-images.githubusercontent.com/55959657/196549366-625fb09b-a449-4bd2-83f0-52833dcf94cf.gif)

The following changes will be added to achieve the behavior shown in the GIF:

- Each button will be enclosed by a `div` tag. This new `div` will have the `data-loading-states` attribute to create different scopes.
- Each button will have an `id` created using the following template string: `btn-{action}-{webhook.id}`. This id will be used to select the right HTML element when the request is in flight. The attribute value for `data-loading-target` can be any valid CSS selector so, if you use a CSS class (like 'btn' or 'btn-danger') every element of that class available in the DOM will be selected and triggered its transition states (I think that's not the behavior you described). It's better to use the `data-loading-target` attribute with id selectors to trigger the neighbor button.


Please let me know what you think.

Reply:
Great! I've reviewed these last changes and it works well! 
Now the spinner is loaded accordingly to the clicked button.

Seems that this feature is completed.
Thank you.